demos/browser/README.md

```diff
@@ -36,6 +36,14 @@ the same AWS region as the meeting and have the following bucket policy:
     ]
 }
 ```
+IVS channel is created in serverless and not in local. To use the Amazon Chime SDK media pipelines live connector feature, you need to provide an Amazon Interactive Video Service (IVS) channel endpoint and set the environment variable when running locally.
```

Comment:
nit: To use the Amazon Chime SDK media pipelines live connector feature, you need to provide an Amazon Interactive Video Service (IVS) channel endpoint and set the environment variable when running locally. IVS channel is created in only in the serverless browser demo and not in local browser demo.